include/bluetooth/direction.h

```diff
@@ -142,6 +142,25 @@ struct bt_df_conn_iq_samples_report {
 	/** Pinter to IQ samples data. */
 	struct bt_hci_le_iq_sample const *sample;
 };
+
+struct bt_df_conn_cte_req_params {
+	/**
+	 * @brief Requested interval for initiating the CTE Request procedure.
+	 *
+	 * Value 0x0 means, run the procedure once. Other values are intervals in number of
+	 * connection events, to run the command periodically.
+	 */
+	uint8_t interval;
+	/** Requested length of the CTE in 8 us units. */
+	uint8_t cte_length;
+	/**
+	 * @brief Requested type of the CTE.
+	 *
+	 * Allowed values are: BT_HCI_LE_AOA_CTE, BT_HCI_LE_AOD_CTE_1US, BT_HCI_LE_AOD_CTE_2US.
+	 */
+	uint8_t cte_type;
```

Comment:
Something to consider, but not completely necessary: We've often `define`d values in the header files rather than relying on the values defined in `hci.h`, see e.g. `gap.h` which mostly consists of such files. 

It could be beneficial to define values such as `BT_DF_AOA_CTE` and `BT_DF_CTE_1US` in this file, so that the user does not need to include `hci.h`.

Reply:
I've added separate commit because there was an `enum by_df_cte_type`. The former one was changed to `enum by_df_cte_type_bits` and all members has added suffix `_BIT` or `_BITS` (for multi-bit value).